storage/innobase/ut/ut0ut.cc

```diff
@@ -519,15 +519,10 @@ const char *ut_strerr(dberr_t num) {
       return ("Sample reader has been requested to stop sampling");
     case DB_OUT_OF_RESOURCES:
       return ("System has run out of resources");
-<<<<<<< HEAD
-
     case DB_PAGE_IS_BLANK:
```

Comment:
This is used by PXB?

Reply:
yes  case DB_PAGE_IS_BLANK:
        return ("Page is blank");
. it was in the previous release as well.